frontend/src/metabase/core/components/ColorRangeSelector/ColorRangePopover.tsx

```diff
@@ -44,30 +44,37 @@ const ColorSelectorContent = forwardRef(function ColorRangeSelector(
     return customColorMapping ?? getDefaultColorMapping(colors);
   }, [colors, customColorMapping]);
 
+  const [isInverted, setIsInverted] = useState(() =>
+    getDefaultIsInverted(initialValue, colorMapping),
+  );
+
   const [color, setColor] = useState(() =>
     getDefaultColor(initialValue, colors, colorMapping),
   );
 
   const [value, setValue] = useState(() =>
-    getColorRange(color, colorMapping, getInverted(initialValue, colorMapping)),
+    getColorRange(color, colorMapping, isInverted),
   );
 
-  const handleSelect = useCallback(
+  const handleColorSelect = useCallback(
```

Comment:
Updated the name of this and a couple other methods to be a bit more descriptive